infection.phar.asc

```diff
@@ -0,0 +1,11 @@
+-----BEGIN PGP SIGNATURE-----
```

Comment:
pls integrate phar.io/phive here 
or if you want I will create an pr for that :-)